### ISSUE TITLE
TB 91.1.2 content view pane layout is now messed up

### ISSUE THREAD

**Issue by AlexHaaniWink** [thunderbird-conversations/thunderbird-conversations#1710]
* **Thunderbird version:** 91.1.2
* **Thunderbird Conversation version:** 3.2.34
* **Operating System:** Ubuntu 21.04
* **Other add-ons:** "No Message Pane Sort" (disabling this makes no change).

**What did you do?**
- I started Thunderbird after it was automatically updated to version 9.1.12 (update of 28 September).
- I looked at e-mails.

**What did you expect to happen?**
E-mail content view looks somewhat normal.

**What actually happened?**
- There's a bar at the top of the content view that doesn't fill the complete width of the panel (normally I run TB fullscreen at 1920x1080, screenshot is simply smaller to reduce file size).
- There's a distracting border with 'padding' and 'margin' around the e-mail content.
  - Padding and margin around the border are even more clear when viewing an HTML e-mail.
- There's always a vertical scrollbar on the right, even when the content fits.

Ps. In the 'enabled' view the two e-mails (sent from and to the same address) are not in a thread anymore because I moved them to a different folder.

![email-pane-with-conversations-enabled](https://user-images.githubusercontent.com/13114801/135226978-b24c0bae-a856-49f5-9872-0d021bfff974.png)
![email-pane-with-conversations-disabled](https://user-images.githubusercontent.com/13114801/135226980-d00d2ae3-24ce-431b-b1a8-85039d1711a1.png)

**Comment by Standard8** [comment]
Can you check what it looks like with a light mode theme rather than a dark one?

Also, check that you don't have a userChrome.css file in your profile (I think it'll be mentioned on Help -> Troubleshooting information if you have one).

There's nothing platform specific afaik, but maybe something's changed in Thunderbird to mess this up.

**Comment by AlexHaaniWink** [comment]
Oh whoops. I indeed have a little bit of `userContent.css`. I guess some parts have been changed related to that.

Removing that fixes the width issue.
Now I'm wondering whether that border is new, or has been there in the past too. Same for the vertical scrollbar.
If so, then this issue can be closed of course. Thanks!

New screenshot with light theme enabled:
![light theme](https://user-images.githubusercontent.com/13114801/135253069-f8b308ba-f9bd-402a-a257-2b170de25884.png)
